kafka-rest/src/main/java/io/confluent/kafkarest/Errors.java

```diff
@@ -51,6 +51,9 @@ public static RestException partitionNotFoundException() {
   public static final String CONSUMER_INSTANCE_NOT_FOUND_MESSAGE = "Consumer instance not found.";
   public static final int CONSUMER_INSTANCE_NOT_FOUND_ERROR_CODE = 40403;
 
+  public static final String CONSUMER_GROUP_ID_NOT_FOUND_MESSAGE = "Consumer group id not found.";
+  public static final int CONSUMER_GROUP_ID_NOT_FOUND_ERROR_CODE = 40403;
```

Comment:
can you use unique numeric id? say 40405

Reply:
Done